package.json

```diff
@@ -9,63 +9,48 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "core-js": "^3.6.5",
-    "vue": "^2.6.11"
-  },
```

Comment:
Is there a reason you took out these things?

Reply:
Sorry this was supposed to be in regards to lines 15-27, not the ones shown above. It's stuff like repository, keywords, author, etc...